pkg/controller/clusterpool/clusterpool_controller_test.go

```diff
@@ -46,6 +47,8 @@ func TestReconcileClusterPool(t *testing.T) {
 	corev1.AddToScheme(scheme)
 	rbacv1.AddToScheme(scheme)
 
+	initialPoolVersion := "21a89f16034caf029cebda7cced933a42d4d5b548134005dcbf29ce484222325"
```

Comment:
maybe we should include some comments on how to update it when we have to change the poolBuilder.